src/app/app.routing.ts

```diff
@@ -3,11 +3,13 @@ import { Routes, RouterModule } from '@angular/router';
 
 import { AuthGuard, UnauthGuard } from 'ngx-prx-styleguide';
 import { ErrorComponent } from './error';
+import { AuthorizationComponent } from './authorization/authorization.component';
 import { HomeComponent, HomeSeriesComponent, HomeStoryComponent } from './home';
 import { LoginComponent } from './login/login.component';
 
 export const routes: Routes = [
   { path: '_error_',   component: ErrorComponent },
+  { path: 'permission-denied', component: AuthorizationComponent },
```

Comment:
Nitpick: could give this a canActivate, similar to `login`, which redirects you back to the root when not applicable.  But in real life ... not likely to happen.

Reply:
Well that's odd!  Yeah, don't worry about it then.